app/views/layouts/_header.html.slim

```diff
@@ -14,4 +14,6 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
             = link_to('Logout', destroy_user_session_path, :method => :delete)
         - else
           li
-            = link_to('Login', new_user_session_path)
\ No newline at end of file
+            = link_to('Login', new_user_session_path)
+      .col-md-12
+        = render_breadcrumbs :separator => " / "
```

Comment:
2 spaces per indentation level

I think it's bad to use breadcrumbs in header. In our case it's not readeble
![01](https://cloud.githubusercontent.com/assets/15616219/11855033/718688da-a450-11e5-81b2-6e74bd8ede1f.jpg)